src/openfl/_internal/renderer/canvas/CanvasGraphics.hx

```diff
@@ -1439,6 +1426,7 @@ class CanvasGraphics {
 				
 				data.destroy ();
 				graphics.__bitmap = BitmapData.fromCanvas (graphics.__canvas);
+				@:privateAccess graphics.__bitmap.__scale = scale;
```

Comment:
Again, `@:privateAccess` shouldn't be required because we have `@:access(openfl.display.BitmapData)` already?